hs_app_timeseries/tests/test_timeseries_metadata.py

```diff
@@ -704,6 +707,7 @@ def test_metadata_is_dirty_flag(self):
         title = self.resTimeSeries.metadata.title
         self.resTimeSeries.metadata.update_element('title', title.id, value="New Resource Title")
         # at this point the is_dirty be set to true
+        self.resTimeSeries.metadata.refresh_from_db()
```

Comment:
Here is a reference to look at to undersand why `refresh_from_db()` is needed. In this case if we don't do refresh then the following statement (self.assertEqual(self.resTimeSeries.metadata.is_dirty, True) ) will fail.